test/integration/targets/meraki_network/tasks/main.yml

```
@@ -56,7 +56,41 @@
       type: appliance
     delegate_to: localhost
     register: create_net_appliance_no_tz
-    
+
+  - name: Enable VLAN support on appliance network
+    meraki_network:
+      auth_key: '{{ auth_key }}'
+      state: present
+      org_name: '{{test_org_name}}'
```

We tend to leave spaces around Jinja statements. We think it helps with readability, but that's also part of what you are used to, and we have been doing it like this for a long time. Although not everyone has been doing it consistently.
```suggestion
      org_name: '{{ test_org_name }}'
```